script-base.js

```diff
@@ -73,15 +74,53 @@ var ScriptBase = yeoman.generators.NamedBase.extend({
   },
 
   _canGenerateTests: function () {
-    return this.config.get('testFramework') === 'mocha' && !this.config.get('includeRequireJS');
+    return this.config.get('testFramework') === 'mocha';
   },
 
-  _generateTest: function (type) {
+  _getMochaUi: function() {
+    var ui = this.config.get('ui');
+
+    if (!ui) {
+      try {
+        ui = require(path.join(process.cwd(), '.yo-rc.json'))['generator-mocha'].ui;
```

Comment:
I'm ok with this, the value provided by composing the generator-mocha in becomes pretty limited though after this